### ISSUE TITLE
Timeout y crash del site cuando el api demora mas de lo normal

### ISSUE THREAD

**Issue by alvarogabrielgomez** [ielijose/actas-elecciones-2024#1]
Existe un problema al momento de buscar algunas actas, con algunos numeros de cedula el site crashea totalmente: 

"Application error: a client-side exception has occurred (see the browser console for more information)."

![image](https://github.com/user-attachments/assets/49181392-7dbd-4f0a-9b25-577434786139)

No sucede siempre, pero parece totalmente un problema de timeout.

**Comment by ReylanLugo** [comment]
Creo que el endpoint que usaba la web fue dado de baja, la web oficial ahora usa otro y da una respuesta distinta

**Comment by alvarogabrielgomez** [comment]
@ReylanLugo Efectivamente, ahora usa un URL de Amazon y la busqueda es por query parameter
https://tvtcrhau2vo336qa5r66p3bygy0hazyk.lambda-url.us-east-1.on.aws/?cedula=
Ahora ya no tiene tantas informaciones como antes, solo el url del acta.
